### ISSUE TITLE
Error when running the example code provided in the blog post with docker environment

### ISSUE THREAD

**Issue by alehresmann** [AethliosIK/reset-tolkien#7]
Hello,

I attempted to run your example from your blog post [here](https://www.aeth.cc/public/Article-Reset-Tolkien/secret-time-based-article-en.html):

I copied the section from [V - Practice](https://www.aeth.cc/public/Article-Reset-Tolkien/secret-time-based-article-en.html#V---Practice)

In that file, I inserted the sample implementation for the `generate_token()` function, described in the same section, where the token is generated with `hashlib.md5(str(t).encode()).hexdigest()`.

Then, I tried to run the scripts provided in section [VI.4 - Practical example](https://www.aeth.cc/public/Article-Reset-Tolkien/secret-time-based-article-en.html#VI4---Practical-example), both for the detection example and the sandwich attack scenario.

The given expected output using the generated command for the detection example is:

```bash
# >> exploit("attacker@example.com")
# $ reset-tolkien detect 2487113242892c39716477efb579538c -d "Wed, 27 Mar 2024 15:10:18 GMT" --prefixes "attacker@example.com" --suffixes "attacker@example.com" --hashes="md5" --decimal-length 6
# The token may be based on a timestamp: 1711552218.352686 (prefix: None / suffix: None)
# The convertion logic is "md5,uniqid"
```

In my case, both scripts to generate the `reset-tolkien` cli input worked, but using the generated cli input resulted in the following error:

```bash
docker run --rm -it --net=host -v "$PWD:/reset-tolkien/" reset-tolkien:latest detect 49a7edd5011a2cd9578f6900588afc4d -d "Thu, 20 Jun 2024 03:02:29 GMT" --prefixes "attacker@example.com" --suffixes "attacker@example.com" --hashes="md5" --decimal-length 6 
The token may be based on a timestamp: 1718852549.408737 (prefix: None / suffix: None) 

Traceback (most recent call last):   
File "/usr/local/bin/reset-tolkien", line 33, in <module>
sys.exit(load_entry_point('reset-tolkien==1.1.1', 'console_scripts', 'reset-tolkien')())
File "/usr/local/lib/python3.10/site-packages/reset_tolkien-1.1.1-py3.10.egg/resetTolkien/cli.py", line 317, in main
formats=Formatter().export_formats(formats)
File "/usr/local/lib/python3.10/site-packages/reset_tolkien-1.1.1-py3.10.egg/resetTolkien/format.py", line 80, in export_formats
raise ValueError(f"No partial name found : {format}")
ValueError: No partial name found : functools.partial(<bound method Formatter.prefix_suffix of <resetTolkien.format.Formatter object at 0x74665d6d1030>>, prefix=None)
```

**Comment by AethliosIK** [comment]
Thanks @alehresmann for this issue.

The error occurs because of the way I retrieve the name of partial functions with a regex. This bug should only appear when using the tool locally, as in your example with the docker.

**Comment by AethliosIK** [comment]
Fixed in #10
